odk1-src/form-functions.rst

```diff
@@ -0,0 +1,719 @@
+.. spelling::
+
+  Anaranjado
+  Azul
+  Colores
+  Español
+  Púrpura
+  Qué
+  Rojo
+  Seleccione
+  arctan
+  arg
+  bday
+  bp
+  br
+  chkn
+  colores
+  fsh
+  gustan
+  jr
+  nodeset
+  prefs
+  seleccionados
+  te
+  tres
+  π
+  
+  
+	
+******************************
+Form Functions and Operators
+******************************
+
+:ref:`expressions` in :ref:`calculations <calculations>`, :ref:`constraints <constraints>`, and :ref:`relevants <relevants>`
+can contain operators and functions.
+
+.. seealso:: `XPath Functions <https://opendatakit.github.io/xforms-spec/#xpath-functions>`_
+
+.. contents::
+  :local:   
+
+  
+.. _xform-operators:
+
+Operators
+==========
+
+.. _utility-operators:
+
+Utility operators
```

Comment:
These are XPath references to the current and containing node. They can be used to build up relative paths (e.g. ../../food). It's ok to put them in this section but I think they should have a little more context. See https://opendatakit.github.io/xforms-spec/#xpath-paths. Perhaps this should go last since it's a little more esoteric.

Reply:
> Perhaps this should go last since it's a little more esoteric

I don't know... in their simple "naive" sense, they get used all the time. You can't really deal with constrains without `.`, and `position(..)` is pretty common inside repeats.

I think we need a doc or section on XPath, separate from this. Once that is in place, I think linking from here to there with a note ("These are actually XPath references.... etc...") would be a good idea. For now, maybe a similar note that links to https://opendatakit.github.io/xforms-spec/#xpath-paths.

Reply:
I'm adding a note, and --- after trying a few arrangements --- ended up putting them last as suggested.